src/luaState.h

```diff
@@ -29,8 +29,8 @@ class LuaState {
 	Variant pullVariant(String name);
 	Variant callFunction(String functionName, Array args);
 
-	Variant getRegistryKey(String name);
-	Ref<LuaError> setRegistryKey(String name, Variant var);
+	Variant getRegistry(String name);
+	Ref<LuaError> setRegistry(String name, Variant var);
```

Comment:
I think get and setRegistryValue can be a little more clear then just registry, as we are also not overwriting the entire registry.

Reply:
....

Reply:
> I think get and setRegistryValue can be a little more clear then just registry, as we are also not overwriting the entire registry.

So now i have to replace it all over again in all 6 files?

Reply:
> So now i have to replace it all over again in all 6 files?

Yes that would be correct. It is a lot easier to do if you don't use the github web editor. Which limits you to 1 file changed per commit. It also would make the PR a lot more readable, as currently its a spam of commits without meaning.

Reply:
> > So now i have to replace it all over again in all 6 files?
> 
> Yes that would be correct. It is a lot easier to do if you don't use the github web editor. Which limits you to 1 file changed per commit. It also would make the PR a lot more readable, as currently its a spam of commits without meaning.

I /did/ set up github ssh keys as you requested, but i dont really know why im not using vscode right now.